soda-core/src/soda_core/contracts/contract_verification.py

```diff
@@ -208,7 +208,12 @@ def log_summary(self, logs: Logs) -> None:
 
     def get_numeric_diagnostic_value(self, diagnostic_name: str) -> any:
```

Comment:
I don't see any usage of this method in this PR or on the main (v4) branch, is it used anywhere?

Reply:
removed it.  good catch.

I moved this one and forgot to delete it.  Also renamed it to shorter version to make tests more readable.